main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"github.com/decred/dcrd/dcrutil"
+)
+
+const appDisplayName = "GoDCR"
+
+var defaultAppDataDir = dcrutil.AppDataDir("godcr", false)
+
+func main() {
+	launchUserInterface(appDisplayName, defaultAppDataDir, "testnet3")
```

Comment:
Does it only work with "testnet3"?

Reply:
we are currently testing on testnet

Reply:
But does the network have to be hard coded in?